src/rinq/ident/ref.go

```diff
@@ -40,8 +40,8 @@ func (ref Ref) After(r Ref) bool {
 	return ref.Rev > r.Rev
 }
 
-// Message returns a new message ID for this ref's revision with seq as the
-// sequence number.
+// Message returns a new message ID derived from this ref's revision with seq as
+// the sequence number.
```

Comment:
This should just say "derived from this ref", not "this ref's revision", minor but more correct.